<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerIGTLConnectorPropertyWidget</class>
 <widget class="QWidget" name="qSlicerIGTLConnectorPropertyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="ConnectorPropertyFrame">
     <property name="title">
      <string>Properties</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="ConnectorNameLabel">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="ConnectorNameEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="ConnectorTypeLabel">
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="ConnectorNotDefinedRadioButton">
          <property name="text">
           <string>Undefined</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="ConnectorServerRadioButton">
          <property name="text">
           <string>Server</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="ConnectorClientRadioButton">
          <property name="text">
           <string>Client</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="ConnectorStateLabel">
        <property name="text">
         <string>Status:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="ConnectorStateCheckBox">
          <property name="text">
           <string>Active</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="PersistentStateCheckBox">
          <property name="text">
           <string>Persistent</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="ConnectorHostnameLabel">
        <property name="text">
         <string>Hostname:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QFrame" name="ConnectorAddressFrame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="ConnectorHostNameEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhUrlCharactersOnly</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ConnectorPortLabel">
           <property name="text">
            <string>  Port:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ConnectorPortEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
